flink-metrics/flink-metrics-slf4j/src/test/java/org/apache/flink/metrics/slf4j/Slf4jReporterTest.java

```diff
@@ -101,16 +96,17 @@ public void testAddGauge() throws Exception {
                         + ": null";
 
         reporter.report();
-        assertThat(testLoggerResource.getMessages(), hasItem(containsString(expectedGaugeReport)));
+        assertThat(testLoggerResource.getMessages())
+                .anyMatch(logOutput -> logOutput.contains(expectedGaugeReport));
     }
 
     @Test
-    public void testAddMeter() throws Exception {
+    void testAddMeter() throws Exception {
         String meterName = "meter";
 
         Meter meter = new MeterView(5);
         reporter.notifyOfAddedMetric(meter, meterName, metricGroup);
-        assertTrue(reporter.getMeters().containsKey(meter));
+        assertThat(reporter.getMeters().containsKey(meter));
```

Comment:
```suggestion
        assertThat(reporter.getMeters()).containsKey(meter);
```

Reply:
oh boy this is gonna be fun to double-check in the future :/